prow/.gitignore

```diff
@@ -13,3 +13,9 @@ cmd/jenkins-operator/jenkins-operator
 cmd/tide/tide
 cmd/clonerefs/clonerefs
 cmd/initupload/initupload
+cmd/phony
```

Comment:
This makes me think you're not using the `bazel` workflow -- which is fine, but at some point you might want to use it, the rest of the repo is very `bazel`-heavy